docs/ops.html

```diff
@@ -3923,6 +3923,10 @@ <h5 class="anchor-heading"><a id="kraft_reconfig_add" class="anchor-link"></a><a
 
   After starting the controller, the replication to the new controller can be monitored using the <code>bin/kafka-metadata-quorum.sh describe --replication</code> command. Once the new controller has caught up to the active controller, it can be added to the cluster using the <code>bin/kafka-metadata-quorum.sh add-controller</code> command.
 
+  The <code>--command-config</code> needs to point to the to-be-added controller's configuration file.
+
+  The <code>controller.quorum.bootstrap.servers</code> property in <code>--command-config</code> file needs to contain an active controller in the quorum.
```

Comment:
perhaps this can be moved down to after L3934 as well.

nit: `Be sure that the <code>controller.quorum.bootstrap.servers</code> property in the specified <code>--command-config</code> file contains at least one controller currently in the quorum.`